recipes/terraform-provider-ssh-tunnel/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set provider = "sshtunnel" %}
+{% set goname = "github.com/johanneswuerbach/terraform-provider-"+provider.lower() %}
+{% set version = "0.2.2" %}
+
+{% set name = goname.split('/')[-1] %}
+
+{% set plugin_dir = os.sep.join(["plugins", "registry.terraform.io", "hashicorp", provider.lower(), version]) %}
+{% set x4_plugin = name + '_v' + version %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://{{ goname }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eeb164e28bcd61f95f0082355e4f7a79f0df7dd66fc28e3f6ef59d477fce4091
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - go build -v -o {{ target_gobin }}..{{ os.sep }}{{ plugin_dir }}{{ os.sep }}{{ target_goos }}_{{ target_goarch }}{{ os.sep }}{{ x4_plugin }}{{ target_goexe }}
```

We should probably use `go-licenses` here:
```suggestion
  script:
    - go-licenses save . --save_path ./thirdparty
    - go build -v -o {{ target_gobin }}..{{ os.sep }}{{ plugin_dir }}{{ os.sep }}{{ target_goos }}_{{ target_goarch }}{{ os.sep }}{{ x4_plugin }}{{ target_goexe }}
```

---

Fixed